Source/LightboxController.swift

```diff
@@ -47,8 +47,12 @@ public class LightboxController: UIViewController {
 
   public private(set) var page = 0 {
     didSet {
+      var result = max(0, page)
+      result = min(images.count - 1, result)
```

Comment:
I think the solution should hold for now, but I just think it's unnecessary to declare a new variable when you can do it directly.